addons/nuxeo-platform-document-routing/nuxeo-routing-rest-api/src/test/java/org/nuxeo/ecm/restapi/server/WorkflowEndpointTest.java

```diff
@@ -512,13 +512,17 @@ public void testSecurityCheckOnGlobalVariable() throws IOException {
         String out = getBodyWithSecurityViolationForStartReviewTaskCompletion(taskId);
         try (CloseableClientResponse response = getResponse(RequestType.PUT, "/task/" + taskId + "/start_review",
                 out)) {
-            // Missing required variables
-            assertEquals(Response.Status.INTERNAL_SERVER_ERROR.getStatusCode(), response.getStatus());
-            JsonNode node = mapper.readTree(response.getEntityInputStream());
-            final String responseEntityType = node.get("entity-type").textValue();
-            final String responseMessage = node.get("message").textValue();
-            assertEquals("exception", responseEntityType);
-            assertEquals("You don't have the permission to set the workflow variable review_result", responseMessage);
+            // Security violation returns OK
+            assertEquals(Response.Status.OK.getStatusCode(), response.getStatus());
+        }
+
+        try (CloseableClientResponse response = getResponse(RequestType.GET,
+                "/workflow/" + createdWorkflowInstanceId)) {
+            assertEquals(Response.Status.OK.getStatusCode(), response.getStatus());
+            JsonNode workflowInstance = mapper.readTree(response.getEntityInputStream());
+            JsonNode variables = workflowInstance.get("variables");
+            // but the global variable has not been modified
+            assertTrue(variables.get("review_result").isNull());
```

Comment:
maybe assert the log was created ?

Reply:
I did not think about it. Do you have any pointer where we already do that?

Reply:
https://github.com/nuxeo/nuxeo-lts/blob/d32456e02f736179cc82d281ac07bd4bdd1b2034/modules/core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestSQLRepositoryQuery.java#L3608